libs/coin-modules/coin-casper/package.json

```diff
@@ -0,0 +1,126 @@
+{
+  "name": "@ledgerhq/coin-casper",
+  "version": "1.3.2",
```

Comment:
Should we ask someome about the version we have to use ? Because recently we always put the same for every coin